modules/ROOT/pages/spike-control-reference.adoc

```diff
@@ -92,9 +82,45 @@ When you apply the policy to your API from the UI, the following parameters are
 | Method & Resource conditions | The option to add configurations to only a select few or all methods and resources of the API | Yes
 |===
 
-The Spike Control policy does not perform quota enforcement. The configuration parameters are restricted to protect the API and backend. Therefore, to ensure maximum performance, configure these parameters to the lowest possible value. 
+To enforce request quotas, see the xref:rate-limiting-sla-based-policy.adoc[Rate-Limiting SLA policy] and xref:rate-limiting.adoc[Rate Limiting policy].
+
+== FAQ
+
+*When does the sliding window start?*
+
+The window starts with the first request after the policy is successfully applied.
+
+*What type of window does the algorithm use?*
+
+The algorithm uses a sliding window by monitoring the transactions for the amount of time for which you configure in the policy.
```

Comment:
Shouldn't this be the first entry rather than the second?

Reply:
```suggestion
*What type of window does the algorithm use?*

The algorithm uses a sliding window that monitors transactions for the amount of time that you configure in the policy.
```

Reply:
Done! Thanks!